public/json/cats.json

```diff
@@ -2,6 +2,7 @@
   {
     "id": 0,
     "imageLink": "https://s3.amazonaws.com/freecodecamp/funny-cat.jpg",
+    "altText": "A white cat wearing a green helmet shaped melon on it's head. ",
```

Comment:
I think I read or watched something that said it was a good idea, so screen readers finish the sentence correctly.